packages/kbn-test/src/failed_tests_reporter/run_failed_tests_reporter_cli.ts

```diff
@@ -88,6 +89,11 @@ export function runFailedTestsReporterCli() {
           absolute: true,
         });
 
+        if (!reportPaths.length && DISABLE_FAILED_TEST_REPORTER) {
```

Comment:
Can you move this somewhere else so that the failed test reporter doesn't run at all if the env var is set? I would expect it to be disabled altogether, not just disabled when there are no reports

Reply:
I actually want to report test failures when there are some. Problem was that if there are no reports (some CI groups have only API tests and are not included in code coverage scope), it was failing not finding results.

Reported test failures can be useful to explain code coverage stats going down (if there are quite many failures ofr some reason), otherwise it can be interpreted like lack of tests.

Reply:
Okay, then it seems like it should be renamed something like `DISABLE_MISSING_TEST_REPORT_ERRORS`. Otherwise, it's named in a confusing way, because it doesn't disable the failed test reporter.